app/services/form_to_resource_service.rb

```diff
@@ -31,16 +31,14 @@ def convert(params, work, current_user)
 
     private
 
-      def process_curator_controlled(params:, work:, current_user:)
+      def process_curator_controlled(params:, work:)
         resource = reset_resource_to_work(work)
-        if current_user.has_role?(:collection_admin, work.collection)
-          resource.doi = params["doi"] if params["doi"].present?
-          resource.ark = params["ark"] if params["ark"].present?
-          resource.version_number = params["version_number"] if params["version_number"].present?
-          resource.collection_tags = params["collection_tags"].split(",").map(&:strip) if params["collection_tags"]
-          resource.resource_type = params["resource_type"] if params["resource_type"]
-          resource.resource_type_general = params["resource_type_general"] if params["resource_type_general"]
-        end
+        resource.doi = params["doi"] if params["doi"].present?
+        resource.ark = params["ark"] if params["ark"].present?
+        resource.version_number = params["version_number"] if params["version_number"].present?
+        resource.collection_tags = params["collection_tags"].split(",").map(&:strip) if params["collection_tags"]
+        resource.resource_type = params["resource_type"] if params["resource_type"]
+        resource.resource_type_general = params["resource_type_general"].to_sym if params["resource_type_general"]
```

Comment:
```suggestion
        resource.resource_type_general = params["resource_type_general"]&.to_sym
```
Should be equivalent?